README.md

```diff
@@ -1,2 +1,20 @@
 # byte_le_engine
 Base game engine for use in NDACM Byte-le Royale games.
+
+## How to run
+
+.\build.bat - will build your code (compile, pretty much)
+
+python .\launcher.pyz g - will generate a map 
+
+python .\launcher.pyz r - will run the game
+
+## Previous Byte-le Competitions
+
+2018 - Dungeon Delvers - https://github.com/jghibiki/Byte-le-Royale-2018
+
+2019 - Space Denizen - https://github.com/topoftheyear/Byte-le-Royale-2019
+
+2020 - Disaster Dispatcher - https://github.com/topoftheyear/byte_le_royale_2020
```

Comment:
2020 is incorrect:
https://github.com/PixPanz/byte_le_royale_2020

Reply:
Fixed